src/transformers/models/maskformer/__init__.py

```diff
@@ -73,7 +73,7 @@
     else:
         from .modeling_maskformer import (
             MASKFORMER_PRETRAINED_MODEL_ARCHIVE_LIST,
-            MaskFormerForInstanceSegmentation,
+            MaskFormerForUniversalSegmentation,
```

Comment:
Same here, shouldn't we keep the old naming import for now?

```suggestion
            MaskFormerForInstanceSegmentation,
            MaskFormerForUniversalSegmentation,
```